yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -871,11 +891,17 @@ module ietf-yang-semver {
     backwards-compatible way.  Two new individual drafts are proposed to go about adding the capabilities in
     different ways: draft-jdoe-netmod-exmod-enhancements and draft-jadoe-netmod-exmod-changes.  These are initially
     developed in parallel with the following versions.</t>
-<t keepWithNext="true">Parallel development for next module revision:</t>
+<t keepWithNext="true">Parallel development for next module revision (track 1):</t>
 <artwork name="" type="" align="left" alt=""><![CDATA[
-      1.1.0-draft-jdoe-netmod-exmod-enhancements-00 || 1.1.0-draft-jadoe-netmod-exmod-changes-00
-        |                                                |
-      1.1.0-draft-jdoe-netmod-exmod-enhancements-01 || 1.1.0-draft-jadoe-netmod-exmod-changes-01
+      T1: 1.1.0-draft-jdoe-netmod-exmod-enhancements-00
+        |
+      T1: 1.1.0-draft-jdoe-netmod-exmod-enhancements-01
+]]></artwork>
+<t keepWithNext="true">In parallel with (track 2):</t>
+<artwork name="" type="" align="left" alt=""><![CDATA[
+      T2: 1.1.0-draft-jadoe-netmod-exmod-changes-00
+        |
+      T2: 1.1.0-draft-jadoe-netmod-exmod-changes-01
 ]]></artwork>
```

Comment:
I think maybe we got the T1/T2 slightly wrong here ?  Shouldn't the first two lines be T1 then T2 (for jdoe) and then similarly T1 then T2 for jadoe ?

Also - the difference between jdoe and jadoe is too subtle. How about jdoe and some totally different name like "fredsmith" ?

Reply:
When I suggested T0, T1 I was thinking *time*. I see you were thinking "Track".  They are already sorted by track (with your new layout) so I'd vote for time, but it seems we should explain what T0, T1 are :-)

Reply:
I think time is already conveyed (i.e., one rev after the other).  It was the move from side-by-side to stacked that I didn't know conveyed track well enough.  I get your point that Tn implies time in a lot of cases.  I think I'll yank it out and change the name to be clearer.